include/seqan3/argument_parser/validators.hpp

```diff
@@ -485,6 +485,82 @@ struct default_validator
     }
 };
 
+//!\brief A helper struct to chain validators recursively via operator&()
+//!\ingroup validators
+template <validator_concept validator_lhs_type, validator_concept validator_rhs_type>
+    requires std::Same<typename validator_lhs_type::value_type, typename validator_rhs_type::value_type>
+struct validator_chain_adaptor
+{
+    //!\brief Type of value that are tested by each of the validators.
+    using value_type = typename validator_lhs_type::value_type;
+
+    /*!\brief Constructing from a vector.
+     * \param[in] lhs_ Some validator to be chained to rhs_.
+     * \param[in] rhs_ Another validator to be chained to lhs_.
```

Comment:
maybe lhs and rhs is confusing as the order does not matter at all. I will change that! 
If the first is false it will throw an exception so no, the second is not evaluated since the program aborts. The "constructing from a vector" is a copy and paste error sorry :see_no_evil: